### ISSUE TITLE
Minor Formatting Error

### ISSUE THREAD

**Issue by Rithenar** [ldtteam/minecolonies#6185]
### Prerequisites 

- [x ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x ] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x ] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.4
- MineColonies Version: 0.13.486-ALPHA-universal
- Structurize Version: 0.13.100-RELEASE-universal
- Related Mods and their Versions:

## Expected behavior
 no formatting errors? :P

## Actual behavior
 sorry to post a link but the screenshot got lost and my computer crashed ¯\_(ツ)_/¯
Basically is just a small formatting error. The name of the builder has no spaces before and after it

https://discord.com/channels/139070364159311872/741531022872674314/783809913465339955

## Steps to reproduce the issue

1. 
2. 
3. 

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Moleculor** [comment]
Does that fix also fix the same issue with Couriers and stash pickup messages also having a similar formatting problem?

**Comment by Raycoms** [comment]
yes
